runtime/shibuya/src/lib.rs

```diff
@@ -7,7 +7,7 @@
 use codec::{Decode, Encode};
 use frame_support::{
     construct_runtime, parameter_types,
-    traits::{Contains, Currency, FindAuthor, Imbalance, Nothing, OnUnbalanced},
+    traits::{Contains, Currency, FindAuthor, Imbalance, Nothing, OnUnbalanced, OnRuntimeUpgrade},
```

Comment:
Please make sure to run `cargo fmt` before push.
Or just push another commit if CI fails.